app/javascript/actions/initialState.js

```diff
@@ -43,9 +43,16 @@ const formatState = {
         ['id', 'name', 'answers', 'building_type_id', 'portfolio_id', 'address', 'city', 'state', 'zip', 'questions']
       ).map((filteredBuilding) => {
         if (filteredBuilding.questions) {
+          const questionIds = Object.keys(filteredBuilding.questions);
+          const editable = questionIds.reduce((map, qId) => {
+            map[qId] = !!filteredBuilding.questions[qId]["can_edit"];
```

Comment:
is `!!` necessary? isn't it just `not not`

Reply:
Changed to `Boolean(...)`!